vedro_allure_reporter/_allure_reporter.py

```diff
@@ -240,3 +270,6 @@ class AllureReporter(PluginConfig):
 
     # Add custom labels to each scenario
     labels: List[Label] = []
+
+    # Run tests by specific allure label
+    allure_labels: [List[str]] = []
```

Comment:
I believe it would be prudent to only allow the use of the --allure-labels option through the command line.